src/features/surveys/components/surveyForm/SurveyFooter.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react';
+import { Box, List, ListItem, Typography } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import SurveyContainer from './SurveyContainer';
+import theme from 'theme';
+
+const links = [
+  {
+    href: 'https://zetkin.org/',
+    text: messageIds.surveyFooter.links.foundation,
+  },
+  {
+    href: 'https://manual.zetkin.org/',
+    text: messageIds.surveyFooter.links.manual,
+  },
```

Comment:
Let's get rid of this. It hasn't been updated in a very long time and is not relevant for Gen3. We will be creating a new knowledge base on our new website, but that's still work in progress.

Reply:
✅ https://github.com/zetkin/app.zetkin.org/pull/2447/commits/77fec5bc133191e4b088510d85f13e6f4ea2301c